cmd/clusterctl/client/config/reader_viper_test.go

```diff
@@ -125,11 +125,10 @@ func Test_viperReader_Init(t *testing.T) {
 func Test_viperReader_Get(t *testing.T) {
 	g := NewWithT(t)
 
-	dir, err := os.MkdirTemp("", "clusterctl")
-	g.Expect(err).ToNot(HaveOccurred())
+	dir := t.TempDir()
 	defer os.RemoveAll(dir)
```

Comment:
```suggestion
```

/hold

Reply:
@sbueringer 
Thanks, I commited this suggestion.